### PR TITLE
Add tweets scrape

### LINKED ISSUE #443: Scrape tweets from twitter accounts
[✓] I am running the latest version

[✓] I checked to make sure that this issue has not already been filed

# Expected Behavior

Scrape the tweets from a given twitter account name by using Twitter API and save this data in a elasticsearch index.

# Current Behavior

Tweets data are not scraping using Twitter API and data are not saved in a elasticsearch server.

### PR DESCRIPTION
**Pull request by JanukaD**
Created a python script for scraping the data such as tweets ID, tweet, text, created time of the tweet and author of the tweet from a given twitter screen name (eg: adaderana) by using Twitter API. It was configured as a cron job by using python setInterval() method and extracted data are saved in a elasticsearch index.

**Closing issues**

closes #443

### CODE REVIEW

File: Crawlers/twitter_accounts/adaderana.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+from elasticsearch import Elasticsearch, helpers
+from threading import Thread, Timer
+from datetime import datetime
+
+
+# Twitter API credentials
+consumer_key = "d4ziZbNtdOzBFnse8rux2iLpa"
+consumer_secret = "1104uapeYVVti40iLyehCms9nbBOkmihZJH8qujKBPRoO3E7G0"
+access_key = "858674738490724354-1BJJJXY3pMCzvo99PUO9600ppy2jsqI"
+access_secret = "f6wscyFUF0NbFNUwGn2DPHecrjIAHc4ru99LN6juQtZRB"
```
Comment (root5533) [inline Crawlers/twitter_accounts/adaderana.py]: Move these keys to a seperate file like an env file

File: Crawlers/twitter_accounts/adaderana.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
+import tweepy
+from elasticsearch import Elasticsearch, helpers
+from threading import Thread, Timer
+from datetime import datetime
```
Comment (root5533) [inline Crawlers/twitter_accounts/adaderana.py]: Have these libraries been added to requirements.txt ? Tweepy was not there.

**Comment by root5533** [comment]
What is the limit of these been crawled ? How can we set this ?

File: Crawlers/twitter_accounts/adaderana.py (hunk 1 above)
Comment (JanukaD) [reply Crawlers/twitter_accounts/adaderana.py]: I have added these keys to a separate dotenv file.

File: Crawlers/twitter_accounts/adaderana.py (hunk 2 above)
Comment (JanukaD) [reply Crawlers/twitter_accounts/adaderana.py]: Added it.

**Comment by JanukaD** [comment]
> What is the limit of these been crawled ? How can we set this ?
> @JanukaD

@root5533 
By using the tweepy python library can scrape the past 3200 tweets from a given user account

**Comment by root5533** [comment]
@JanukaD let's have this limit or count for amount of scraping tweets as a value in env file so we can easily adjust it.

**Comment by root5533** [comment]
Finally squash all the commits into one and update PR.

**Pull request merged**